src/components/Questions/Body.vue

```diff
@@ -365,6 +371,16 @@ export default defineComponent({
 
     if (isQuestionImagePresent.value) startImageLoading();
 
+    onMounted(() => {
+      // Force render any math on the page when component is mounted
+      if ('MathJax' in window) (window.MathJax as any).typeset();
```

Comment:
when would it not be in `window`?

Reply:
This check is applied so it doesn't fail in tests
Try removing `if (`MathJax` in window)` and you'll understand why I've put it here